ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installer-rehearse-4.20.yaml

```diff
@@ -42,34 +42,13 @@ resources:
       memory: 200Mi
 tests:
 - as: installer-rehearse-aws
-  cron: '@yearly'
+  cron: '@monthly'
```

Comment:
This job is for a one-time test only, so please don't change cron config.

Reply:
Same reason as above, this PR is for one-time triggered testing and won't be merged.